server/aws-lsp-codewhisperer/src/language-server/netTransform/transformHandler.ts

```diff
@@ -308,28 +307,31 @@ export class TransformHandler {
                     transformationJobId: request.TransformationJobId,
                 } as GetTransformationRequest
                 this.logging.log(
-                    'poll : send request to get transform  api: ' + JSON.stringify(getCodeTransformationRequest)
+                    'polling details: Sending request to get transform  api: ' +
+                        JSON.stringify(getCodeTransformationRequest)
                 )
                 response = await this.client.codeModernizerGetCodeTransformation(getCodeTransformationRequest)
-                this.logging.log('poll : received response from get transform  api: ' + JSON.stringify(response))
-                this.logging.log('poll : job status here : ' + response.transformationJob.status)
+                this.logging.log(
+                    'polling details: Received response from get transform  api: ' + JSON.stringify(response)
+                )
+                this.logging.log('polling details: Current job status is ' + response.transformationJob.status)
 
                 if (response.transformationJob?.status) {
                     this.logging.log(
-                        'status is included in validExitSTatus for poll ' +
+                        'Status matches to the desired final status of polling request' +
                             validExitStatus.includes(response.transformationJob.status)
                     )
                 }
 
                 if (validExitStatus.includes(status)) {
-                    this.logging.log('returning status as : ' + status)
+                    this.logging.log('Exiting polling loop with job status as : ' + status)
```

Comment:
Reads as a debug log. Can we tighten this? Anytime we exit this polling loop, there should be a log indicating that the work here is complete, whether the job failed, was canceled, or completed successfully. We shouldn't need to explicitly state it unless it has meaning to the customer.